src/components/Blog/Blog.js

```diff
@@ -28,6 +37,10 @@ export default function Blog({style}) {
               )
             )}
           </div>
+          <BackButton className="tag-and-back" onClick={() => navigate(-1)}>
```

Comment:
I think you don't need the className here and anywhere else.
```suggestion
          <BackButton onClick={() => navigate(-1)}>
```

Reply:
Well, I see why you do it, but it still seems strange to me to mix styled components with the className attribute.